config/ibm.words

```diff
@@ -357,7 +357,7 @@ reserved:	PLUS
 reserved:	POINTER
 #reserved:	POINTER-24       # potentially reserved
 #reserved:	POINTER-31       # potentially reserved
-reserved:	POINTER-32 X
+reserved:	POINTER-32
```

Comment:
nice that you've spot the typo, the change was so small that I've directly checked that in (if I remember right the target branch will be updated tomorrow)